mobify/stencil-select
# Get ui working in tests, use “class”-based convention for ui.

Gets the component’s ui script working in the test env. Also refactors the script to use a class/prototype-based approach that feels more comfortable to me for managing instances. This exposes the component’s instance at `$('.c-select').data('component')`, and emits a `select:update` event when the select updates itself.

Status: **Opened for review**

Reviewers: @fractaltheory @donnielrt 
Ticket: N/A
Linked PRs: N/A
## Changes
- Update adaptive devDep
- Add jQuery as new devDep
- update Select ui to use jQuery, refactor to use a “class”-based approach
- update test runner to run Select ui
## How to test-drive this PR
- Check out this branch
- Delete node_modules and do a fresh `npm install`
- Run `grunt` and look for the port the server is using
- Open `http://localhost:{port}/tests/visual`
- Change the value of the third select instance and ensure it updates
## Research resources
- none
